packages/styletron-core/src/index.js

```diff
@@ -6,15 +6,17 @@ class StyletronCore {
 
   /**
    * Create a new StyletronCore instance
-   * @param {object} [opts]           An object containing options
-   * @param {string} [opts.prefix=''] A prefix for generated CSS class names
+   * @param {object} [opts]                           An object containing options
+   * @param {string} [opts.prefix='']                 A prefix for generated CSS class names
+   * @param {boolean} [opts.readableClassNames=false] Use readable class names
    */
-  constructor({prefix = ''} = {}) {
+  constructor({prefix = '', readableClassNames = false} = {}) {
```

Comment:
A late preference is expressed: `expandClassNames = false` by default, or `expandClassNames = true` as a manual override during development.

I prefer the command form ("expand" as opposed to "expanded"). And I'm not fond of "readable", since they're only marginally readable :) 

Also, the semantics of compression-by-default is important. I'd prefer to steer clear of anything that names the behavior in any way other than the default (e.g., "sequential" or "compressed"). 

Carry on, and I love the idea! Will use during development for sure.